apps/console/src/features/connections/components/edit/forms/helpers/form-fields-helper.tsx

```diff
@@ -178,6 +178,50 @@ export const getRadioButtonField = (eachProp: CommonPluggableComponentPropertyIn
     );
 };
 
+// Add a radio field for user id in claims with predefined options.
```

Comment:
Better to have a TSDoc comment.

Reply:
Improved with 520dba0695d501fe0ca61d7edb365d58ea7cf899.